webpack/assets/javascripts/react_app/components/common/DocumentationLink/__snapshots__/DocumentationLink.test.js.snap

```diff
@@ -14,6 +14,7 @@ exports[`documentation links should have an external link to documentation 1`] =
     name="question-circle"
     type="fa"
   />
-   Documentation
+   $
+  Documentation
```

Comment:
Looks like this was missed

Reply:
https://projects.theforeman.org/issues/32656 yup already created it here